Makefile

```
@@ -116,6 +116,21 @@ lint-fix: $(GOLANGCI_LINT_BIN)
+$(GOVULNCHECK_BIN): $(GO)
+	GOFLAGS= GOBIN=$(BIN_DIR) $(GO) install golang.org/x/vuln/cmd/govulncheck@latest
+
+govulncheck: $(GOVULNCHECK_BIN)
+	@$(GOVULNCHECK_BIN) -format json ./...
+
+GOVULNCHECK_PARSER_BIN := $(BIN_DIR)/govulncheck-parser
+
+$(GOVULNCHECK_PARSER_BIN): $(GO)
+	$(GO) build -o $(GOVULNCHECK_PARSER_BIN) ./tools/govulncheck-parser
+
+govulncheck-parser: $(GOVULNCHECK_PARSER_BIN)
```

checkmake (0.3.2)

[warning] Target "govulncheck-parser" should be declared PHONY.

(phonydeclared)